src/worker/vm/extensions/retry.ts

```diff
@@ -0,0 +1,23 @@
+import { PluginConfig, PluginsServer } from '../../../types'
+
+const minRetry = process.env.NODE_ENV === 'test' ? 1 : 30
+
+// TODO: add type to scaffold
+export function createRetry(
+    server: PluginsServer,
+    pluginConfig: PluginConfig
+): (type: string, payload: any, retry_in?: number) => Promise<void> {
+    return async (type: string, payload: any, retry_in = 30) => {
+        if (retry_in < minRetry || retry_in > 86400) {
```

Comment:
Agreed, though can we keep as is? https://github.com/PostHog/plugin-server/pull/351 will refactor this totally. We'll get into bad merge conflicts otherwise...